camus-etl-kafka/src/test/java/com/linkedin/camus/etl/kafka/CamusJobTestWithMock.java

```diff
@@ -207,26 +216,27 @@ private FetchResponse mockFetchResponse(List<MyMessage> myMessages) {
     }
     ByteBufferMessageSet messageSet = new ByteBufferMessageSet(messages);
     EasyMock.expect(fetchResponse.messageSet(EasyMock.anyString(), EasyMock.anyInt())).andReturn(messageSet).times(1);
-    _mocks.add(fetchResponse);
+    mocks.add(fetchResponse);
     return fetchResponse;
   }
 
-  private void mockSimpleConsumer(TopicMetadataResponse metadataResponse, OffsetResponse offsetResponse,
+  private SimpleConsumer mockSimpleConsumer(TopicMetadataResponse metadataResponse, OffsetResponse offsetResponse,
```

Comment:
The method now returns SimpleConsumer rather than void, because the return value needs to be passed to EtlInputFormatForUnitTest.consumer